packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -0,0 +1,213 @@
+          <oc-icon name="group" />
+        </oc-button>
+        <oc-button
+          v-if="hasLinkShares"
+          v-oc-tooltip="linksLabel"
+          appearance="raw"
+          class="oc-mr-xs"
+          :aria-label="linksLabel"
+          @click="expandLinksAccordion()"
+        >
+          <oc-icon name="link" />
+        </oc-button>
+        <p class="oc-my-rm oc-mx-s" v-text="detailSharingInformation" />
+      </div>
+      <table class="details-table" aria-label="detailsTableLabel">
+        <tr data-test-id="lastModified">
+          <th class="oc-pr-s" v-text="lastModifiedTitle" />
+          <td>
+            <oc-button
+              v-if="versions.length > 0 && !isPublicPage"
+              v-oc-tooltip="seeVersionsLabel"
+              appearance="raw"
+              :aria-label="seeVersionsLabel"
+              @click="expandVersionsAccordion()"
+              v-text="capitalizedModifiedDate"
+            />
+            <span v-else v-text="capitalizedModifiedDate" />
+          </td>
+        </tr>
+        <tr v-if="highlightedFile.ownerDisplayName" data-test-id="ownerName">
+          <th class="oc-pr-s" v-text="ownerTitle" />
+          <td>
+            <p class="oc-m-rm">
+              {{ highlightedFile.ownerDisplayName }}
+              <span v-if="isCurrentUser" v-translate>(me)</span>
+            </p>
+          </td>
+        </tr>
+        <tr data-test-id="sizeInfo">
+          <th class="oc-pr-s" v-text="sizeTitle" />
+          <td v-text="getResourceSize(highlightedFile.size)" />
+        </tr>
+        <tr v-if="versions.length > 0 && !isPublicPage" data-test-id="versionsInfo">
+          <th class="oc-pr-s" v-text="versionsTitle" />
+          <td>
+            <oc-button
+              v-oc-tooltip="seeVersionsLabel"
+              appearance="raw"
+              :aria-label="seeVersionsLabel"
+              @click="expandVersionsAccordion()"
+              v-text="versions.length"
+            />
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+</template>
+<script>
+import Mixins from '../../../mixins'
+import MixinResources from '../../../mixins/resources'
+import MixinRoutes from '../../../mixins/routes'
+import { mapActions, mapGetters, mapMutations } from 'vuex'
+import { ImageDimension, ImageType } from '../../../constants'
+
+export default {
+  mixins: [Mixins, MixinResources, MixinRoutes],
+  title: $gettext => {
+    return $gettext('Details')
+  },
+  data: () => ({
+    loading: false
+  }),
+  computed: {
+    ...mapGetters('Files', ['highlightedFile', 'versions']),
+    ...mapGetters(['getToken']),
+    ...mapGetters(['user']),
+
+    detailSharingInformation() {
+      return this.$gettext(
+        'This file has been shared. Manage access rights by clicking on the icons on the left or on the sections below'
+      )
+    },
+    detailsTableLabel() {
+      return this.$gettext('Overview on the information about the selected file')
+    },
+    sharesLabel() {
```

Comment:
```suggestion
    peopleSharesLabel() {
```
for consistency with other computed props